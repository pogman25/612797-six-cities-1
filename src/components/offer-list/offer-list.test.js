import React from 'react';
import renderer from 'react-test-renderer';
import OfferCardsList from './offer-list.jsx';

it(`OfferCardsList correctly renders after relaunch`, () => {
  const tree = renderer
  .create(<OfferCardsList
    offers= {[]}
    onCardHeaderClick={jest.fn()}
  />)
  .toJSON();

  expect(tree).toMatchSnapshot();
});
