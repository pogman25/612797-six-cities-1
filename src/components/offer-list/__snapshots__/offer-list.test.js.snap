// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferCardsList correctly renders after relaunch 1`] = `
<div
  className="cities__places-list places__list tabs__content"
/>
`;
